events/2019/05-contributor-summit/README.md

```diff
@@ -21,10 +21,72 @@ In some sense, the summit is a real-life extension of the community meetings and
 The Contributor Summit takes place in the days leading up to [KubeCon + CloudNativeCon Europe](https://events.linuxfoundation.org/events/kubecon-cloudnativecon-europe-2019/), make sure you plan travel accordingly.
 
  - Sunday, May 19, 2019 - Social event (optional)
- - Monday, May 20, 2019 - Contributor Summit
+ - Monday, May 20, 2019 - Contributor Summit (New Contributor Workshops and SIG F2F meetings)
  - Fira Gran Via, Barcelona, Spain
  - Event website with more information: LINK TBD
 
+## New Contributor Workshops
+
+The focus of the contributor summit in Barcelona is on new contributors. We have a large existing contributor base, especially in the US, and while we have a solid contingent in Europe, we would like to continue to grow our contributor presence in Europe to have a better balance across time zones and a more diverse set of contributors.
+
+**Important**: The new contributor workshops are for people who are interested in **contributing** to Kubernetes, and **will not** include how to use / do k8s. The rest of KubeCon will have plenty of content about using k8s.
+
+### The New Contributor Workshops will have 2 tracks:
+
+* 101 New Contributors: audience is brand new contributors who are interested in contributing, but have little to no experience contributing to open source projects.
+* 201 Intermediate New Contributors: audience is people who have experience contributing to other open source projects, but who are new to contributing to k8s.
+
+**Note**: Existing contributors should **not** register for either of these workshops, but you are welcome to attend the social event on Sunday, and you should talk to your SIG chairs to see if they plan to have a SIG F2F meeting on Monday.
+
+### Key Facts:
+
+* We are expecting 200 people with 100 of those attending the 101 New Contributor Workshop and 100 of those attending the 201 Intermediate New Contributor Workshop. 
+* We have space for 140 people in room 4.1, and 100 people in room 5.1.
```

Comment:
Is this detail needed here?

Reply:
I wasn't sure, but looking at it again, I think we can delete it.